src/main/java/com/aws/greengrass/clientdevices/auth/certificate/events/SecurityConfigurationChanged.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.certificate.events;
```

Comment:
nit: seems like more of an iot domain thing to me?

Reply:
All this actually should go on the configuration domain we have. I think the iot and certificate "domain" would become more of of the authentication domain (maybe not?)